search.go

```diff
@@ -134,19 +159,23 @@ func validKibanaVersion(version *semver.Version, m *util.Manifest) (bool, error)
 	return true, nil
 }
 
-func servePackages(packagesList map[string]*util.Manifest, w http.ResponseWriter) ([]byte, error) {
+func servePackages(packagesList map[string]map[string]*util.Manifest, w http.ResponseWriter) ([]byte, error) {
 
 	// Packages need to be sorted to be always outputted in the same order
 	var keys []string
-	for k := range packagesList {
-		keys = append(keys, k)
+	for key, k := range packagesList {
+		for v := range k {
+			keys = append(keys, key+"-"+v)
+		}
 	}
 	sort.Strings(keys)
 
 	var output []map[string]interface{}
 
 	for _, k := range keys {
-		m := packagesList[k]
+		// TODO: this could break if we have integrations with `-` in the name
```

Comment:
I created [this ticket](https://github.com/elastic/kibana/issues/41253) to change to `name@version` in the Manager for this reason.

Reply:
Good idea, will do the same here.